src/components/ButtonPanel.js

```diff
@@ -10,15 +11,21 @@ const ButtonPanel = () => {
     'group-5': ['0', '.', '='],
   };
 
+  const handleClick = buttonName => {
+    onClick.clickHandler(buttonName);
+  };
```

Comment:
**REQUIRED**
* `onClick` is function not an object. We can re-write this as:
```js
  const handleClick = buttonName => {
    onClick.clickHandler(buttonName);
  };
```

Reply:
Morning Simon. I haven't understood the correction that you have suggested.